### PR TITLE
New License: HPND-export2-US

### LINKED ISSUE #2350: New license request: HPND-export-US-2disclaimers [SPDX-Online-Tools]
**1.** License Name: HPND with US Government export control warning and 2 disclaimers
**2.** Short identifier: HPND-export-US-2disclaimers
**3.** License Author or steward: Unknown
**4.** Comments: This license is similar to HPND-export-US, but it has an additional disclaimer.

This license is used in Fedora, and was found during license review:
https://gitlab.com/fedora/legal/fedora-license-data/-/issues/462
**5.** License Request Url: http://tools.spdx.org/app/license_requests/343
**6.** URL(s): https://github.com/krb5/krb5/blob/krb5-1.21.2-final/NOTICE#L111-L133, https://web.mit.edu/kerberos/krb5-1.21/doc/mitK5license.html
**7.** OSI Status: Not Submitted
**8.** Example Projects: https://web.mit.edu/kerberos/

### PR DESCRIPTION
**Pull request by jlovejoy**
fixes #2350

### CODE REVIEW

**Comment by jlovejoy** [comment]
@swinslow - went ahead and made a PR for this one using my suggested id...

File: src/HPND-export2-US.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="HPND-export2-US" name="HPND with
+   US Government export control and 2 disclaimers" listVersionAdded="3.24">
```
Comment (xsuchy) [inline src/HPND-export2-US.xml:4]: Line break in value of attribute. Please move it to a single line.

**Comment by xsuchy** [comment]
Otherwise LGTM.

File: src/HPND-export2-US.xml (hunk 1 above)
Comment (swinslow) [reply src/HPND-export2-US.xml]: Resolved in bf1a87f, thanks @xsuchy!

**Pull request merged**